docs/api-testing/project-access.md

```diff
@@ -1,66 +1,99 @@
 
-In API Testing, on the Projects page, under a project name, click **Settings**. On the Settings page you can edit project details (Name, Tag(s), Description, and Notes), and view the project's access setting.
+1. In **API Testing**, on the **Projects** page, click **Create Project**. 
+1. Select an option in the **Create from** dropdown. 
+1. Optionally, select one or more **Tags**, or click **Add custom tag** to add a new tag. 
+1. Optionally, enter a **Description** of the project. 
+1. Optionally, enter **Notes** about the project. 
+1. Select the **Access** level for the project. If you are a team member you can select your team or organization-wide access. If you are an organization admin, you can select any team or organization-wide access.  This selection cannot be changed once the project is saved. 
+1. Click **Save**. 
 
-### Access
+### Manage Access to Projects
 
-Access is assigned when you create a project. You can set the project to be accessible to your team (if you're part of one) or to the entire organization.
+Once a project has been created and saved, the access level (team or organization-wide access) is uneditable. 
 
-<img src={useBaseUrl('img/api-fortress/2021/12/accessOrg1.png')} alt="New user UI" width="300" />
+If an organization admin deletes a team from Sauce Labs [Accounts > Team Management](/basics/acct-team-mgmt/managing-user-info/) before managing project access in API Testing, the access level defaults to Organization because the access setting is required. However, the project will still be available only to the original team and does not change to organization-wide access. 
 
-You cannot edit project access once it has been set.
+If you need to change access to a project, you have the following options. 
 
-#### User Scenarios
+#### Update Teams in Sauce Labs Accounts > Team Management
 
-- **New User** (no team assignments or projects): When this user first logs in to API Testing, they will be guided through creating a project and a test.<br/><img src={useBaseUrl('img/api-fortress/2021/12/emptyState.png')} alt="New user UI" width="500" /><br/> They will have the option to set organization-wide project access and will only be able to see the projects they created.
-- **Team Member**: This user belongs to a team and can create and interact with their team's projects and organization-wide projects. They cannot view projects with access restricted to an outside team. When creating a project, they will have the option to set access to be team-specific and organization-wide.
-- **Organization Admin**: This user can create and interact with all data in all projects across their organization.
+If access is assigned to a team, you can add or remove team members from the team in Accounts > Team Management. You must be an organization admin to work with teams in Team Management. 
```

Comment:
@aliciakrebs I would suggest to clearly say that "Accounts > Team Management" is in Sauce Labs dashboard not in APIT